### PR TITLE
Add PeerConnection::ConfigureAudioEncoders

### PR DESCRIPTION
**Pull request by peter-signal**
Allows precise control over OPUS.

### CODE REVIEW

File: modules/audio_coding/codecs/opus/opus_interface.h [hunk 1]
```diff
@@ -526,6 +526,14 @@ int WebRtcOpus_PacketHasFec(const uint8_t* payload,
 int WebRtcOpus_PacketHasVoiceActivity(const uint8_t* payload,
                                       size_t payload_length_bytes);
 
+
+int16_t WebRtcOpus_SetMaxBandwidth(OpusEncInst* inst, int32_t max_bandwidth);
+int16_t WebRtcOpus_SetPredicationDisabled(OpusEncInst* inst, int32_t disable_prediction);
+int16_t WebRtcOpus_SetForceMode(OpusEncInst* inst, int32_t mode);
+int16_t WebRtcOpus_SetSignal(OpusEncInst* inst, int32_t signal);
+int16_t WebRtcOpus_SetApplication(OpusEncInst* inst, int32_t application);
```
Comment (jim-signal) [inline modules/audio_coding/codecs/opus/opus_interface.h:534]: After measurements, it seems that we won't need to change these settings, especially at the lower bitrates we are working with, so maybe remove these changes?

File: media/engine/webrtc_voice_engine.cc [hunk 2]
```diff
@@ -21,6 +21,7 @@
 #include "absl/strings/match.h"
 #include "api/audio_codecs/audio_codec_pair_id.h"
 #include "api/call/audio_sink.h"
+#include "api/peer_connection_interface.h"  //
```
Comment (jim-signal) [inline media/engine/webrtc_voice_engine.cc:24]: Missing comment? Or remove?

File: modules/audio_coding/codecs/opus/opus_interface.h (hunk 1 above)
Comment (jim-signal) [reply modules/audio_coding/codecs/opus/opus_interface.h:534]: That is, prediction, application, signal, and forceMode.

File: media/engine/webrtc_voice_engine.cc (hunk 2 above)
Comment (peter-signal) [reply media/engine/webrtc_voice_engine.cc:24]: I moved it to audio_encoder.h.  I'll remove this.

File: modules/audio_coding/codecs/opus/opus_interface.h (hunk 1 above)
Comment (peter-signal) [reply modules/audio_coding/codecs/opus/opus_interface.h:534]: We don't need MaxBandwidth, either.  I think we just want SetBandwidth.  I'll remove them.

**Review by jim-signal — APPROVED** [review]

**Pull request merged**